cmd/gazelle/metaresolver.go

```diff
@@ -50,12 +54,31 @@ func (mr *metaResolver) MappedKind(pkgRel string, kind config.MappedKind) {
 	mr.mappedKinds[pkgRel] = append(mr.mappedKinds[pkgRel], kind)
 }
 
+// WrapperMacros records the configured wrapper macros for a package
+func (mr *metaResolver) WrapperMacros(pkgRel string, wrapperMacros map[string]string) {
+	// Note: it is somewhat of a hack to store the wrapper macros in the metaResolver
+	// by each package. A more appropriate place for this would be to keep it in the
+	// config.Config struct. However, the config.Config struct is not available at
+	// all of the call sites where the Resolve method is called.
+	//
+	// For example, when the RuleIndex is finalizing and collecting information about
+	// embedded targets, it does this once across the entire index. 
```

Comment:
I have a follow-up that refactors the metaresolver to instead take a `c *config.Config` in the `Resolver` method, which removes this and the above `MappedKind` handling. I decided to keep them separate and follow existing patterns in this PR.